packages/nc-gui/components/account/Token.vue

```diff
@@ -53,12 +75,7 @@ loadTokens()
 const isModalOpen = ref(false)
 const tokenDesc = ref('')
 const tokenToCopy = ref('')
-
-const openModal = (tk: string, desc: string) => {
-  isModalOpen.value = true
-  tokenToCopy.value = tk
-  tokenDesc.value = desc
-}
+const isInvalidInput = ref(false)
```

Comment:
Same problem here. I need to look through the code to figure out what `Input` is referring to. Also please use affirmative naming. I'd suggest to call it as `isValidTokenName`.